examples/nidcpower_source_dc_voltage/_nidcpower_helpers.py

```diff
@@ -0,0 +1,54 @@
+"""nidcpower Helper classes and functions for MeasurementLink examples."""
+
+from typing import Any, Dict, Iterable, Optional
+
+import grpc
+import nidcpower
+
+import ni_measurementlink_service as nims
+
+# To use a physical NI SMU instrument, set this to False or specify
+# --no-use-simulation on the command line.
+USE_SIMULATION = True
+
+
+def create_nidcpower_session(
+    session_info: nims.session_management.SessionInformation,
+    session_grpc_channel: grpc.Channel,
+) -> nidcpower.Session:
+    """Create driver session based on the instrument type and reserved session."""
+    options: Dict[str, Any] = {}
+    if USE_SIMULATION:
+        options["simulate"] = True
+        options["driver_setup"] = {"Model": "4141"}
+
+    session_kwargs: Dict[str, Any] = {}
+
+    session_kwargs["grpc_options"] = nidcpower.GrpcSessionOptions(
+        session_grpc_channel,
+        session_name=session_info.session_name,
+        initialization_behavior=nidcpower.SessionInitializationBehavior.AUTO,
+    )
+
+    return nidcpower.Session(
+        resource_name=session_info.resource_name, options=options, **session_kwargs
+    )
+
+
+def reserve_session(
+    session_management_client: nims.session_management.Client,
+    pin_map_context: nims.session_management.PinMapContext,
+    timeout: int,
```

Comment:
`timeout` is an optional float, not a required int: https://github.com/ni/measurementlink-python/blob/main/ni_measurementlink_service/session_management.py#L173

(You're allowed to pass an int like 60 to a function that expects a float.)

Reply:
Updated the type annotation to `Optional[float]`